source/administration/config-settings.rst

```diff
@@ -4037,6 +4037,102 @@ Changes to this setting require a server restart before taking effect.
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsFileEnabled": []`` with options ``true`` and ``false``.                               |
++-------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+File name
+^^^^^^^^^^^^^^
+This is the path to the output file location.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsFileName": []`` with string array input consisting of a user-defined name.                     |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+File max size MB
+^^^^^^^^^^^^^^
+This is the maximum size (measured in megabytes) that the file can grow before triggering rotation. The default setting is 100.
+
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsFileMaxSizeMB": []`` with numerical input.                                                    |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+File max age days
+^^^^^^^^^^^^^^
+This is the maximum age in days a file can reach before triggering rotation. There is no default limit, and no limit to the value used.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsFileMaxAgeDays": []`` with numerical input.                                                    |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+File max backups
+^^^^^^^^^^^^^^
+This is the maximum number of rotated files kept; oldest is deleted first.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsFileMaxBackups": []`` with numerical input.                                                    |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+File compress
+^^^^^^^^^^^^^^
+This when “true” rotated files are compressed using ``gzip``.
```

Comment:
@wiggin77, what's the definition of "true"? Is it based on any of the other configurations or is it separate?

Reply:
FileCompress is a boolean;  it can be `true` or `false`.   Like the other FileXXX config options, it is only used when `FileEnabled` is `true`.

Reply:
That makes sense, thanks. So then ``FileCompress`` is not numerical, it would be ``true`` or ``false``?

Reply:
Correct